infra/k8s/cluster.yaml

```diff
@@ -93,17 +94,18 @@ spec:
     content: |
       #!/bin/sh
       cat <<EOT >> /etc/sysctl.d/999-testground.conf
-      net.core.somaxconn = 100000
-      net.netfilter.nf_conntrack_max = 1000000
-      net.ipv4.tcp_max_syn_backlog = 100000
-      net.core.netdev_max_backlog = 100000
+      net.core.somaxconn = 131072
+      net.netfilter.nf_conntrack_max = 1048576
+      net.ipv4.tcp_max_syn_backlog = 131072
+      net.core.netdev_max_backlog = 524288
       net.ipv4.ip_local_port_range = 1024 65535
       net.ipv4.tcp_tw_recycle = 1
       net.ipv4.tcp_tw_reuse = 1
-      net.core.rmem_max = 134217728
-      net.core.wmem_max = 134217728
-      net.ipv4.tcp_rmem = 4096 12582912 67108864
-      net.ipv4.tcp_wmem = 4096 12582912 67108864
+      net.core.rmem_max = 131072
+      net.core.wmem_max = 131072
+      net.ipv4.tcp_mem = 262144 524288 1048576
+      net.ipv4.tcp_rmem = 4096 65536 131072
+      net.ipv4.tcp_wmem = 4096 65536 131072
```

Comment:
Should we end these with `sysctl -p` to apply this file?

Reply:
I don't think so, why? They are in the `sysctl.d` directory and they get picked up on reboot.

Reply:
Right, but you wouldn't reboot after this usually.

Reply:
These values are applied to the host, you can check them when you ssh, and then `sysctl -a`.

I don't think we need a reboot, after they've been set in the `userdata`.